### ISSUE TITLE
Hedging in JSON-LD

### ISSUE THREAD

**Issue by bgyori** [clulab/eidos#444]
PR #432 added the extraction of hedgings to mentions with a test sentence ```Rainfall is likely to cause flooding.``` Running this sentence in the shell, I found that it produced the expected Hedging attachment:
```
events: 
List(Causal, DirectedRelation, EntityLinker, Event) => Rainfall is likely to cause flooding 
	 ------------------------------ 
	 Rule => ported_syntax_1_verb-Causal 
	 Type => EventMention 
	 ------------------------------ 
	 trigger => cause 
	 cause (Concept, Entity) => Rainfall 
	  * Attachments: Quantification(likely,None) 
	 effect (Concept, Entity) => flooding 

	 Event Attachments: Hedging(likely) 
	 ------------------------------ 
```
However, I can't find this attachment in the JSON-LD output in any form. Are we sure it appears in the JSON-LD? Sorry if I'm just missing something!

**Comment by kwalcock** [comment]
@bsharpataz, did you see this?  I don't remember it being on my list, but then there are lots of things I don't remember, most of them in fact.

**Comment by bsharpataz** [comment]
closed with #446
